websites/P/PodcastAddict/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "PodcastAddict",
+    "author": {
+      "name": "Harold",
+      "id": "141206144264699904"
+    },
+    "category": "music",
+    "logo": "https://i.imgur.com/tOin3KD.png",
+    "thumbnail": "https://i.imgur.com/IB8PF6i.jpg",
+    "version": "1.0.0",
+    "color": "#fc7e0f",
+    "tags": [
+      "podcast",
+      "music"
+    ],
+    "url": ["podcastaddict.com", "www.podcastaddict.com"],
```

Comment:
You only need one

Reply:
From what I've tested, it needs both for some weird reason, idk

Reply:
No. Site works on this domain and this subdomain too.

Reply:
Oh wait what? Are you talking to me or MareGraphics, ririxi? Today I learnt I guess xD

Reply:
Oh, I had some troubles since it would use url without www subdomain and the presence didn't work, but I'll keep only one with www subdomain

Reply:
When I was writing my comment, I didn't see @MareGraphics comment. Apologize. But I was talking to you @EncryptedDev. Site works either on `podcastaddict.com` and `www.podcastaddict.com` there is no redirect in any way.

Reply:
Ooh, good to know if the future. Ignore my message - sorryy!

Reply:
@ririxidev Yeah that was the problem for me, since it doesn't redirect